### DISCUSSION TITLE
code_action notification on every keypress

### DISCUSSION THREAD

**Discussion by cyaconi** [LazyVim/LazyVim#1178] [Q&A]
I'm getting this annoying notification every time I move my cursor in the current buffer. Any way to get rid of it?

<img width="242" alt="image" src="https://github.com/LazyVim/LazyVim/assets/91154/b3fbb2a1-019f-4682-81a0-e2f3b81acb45">

Thanks!

**Comment by folke** [comment]
Do you have somthing like lspsaga or that lightbulb plugin installed?
Some plugin is fetching coed actions for every line it seeems

**Comment by cyaconi** [comment]
Lspsaga, indeed. Is there a way to configure those notifications?

**Reply by folke** [reply]
There is. Search the noice.nvim issues/discussions.

**Reply by cyaconi** [reply]
Awesome, I didn't know it was noice the culprit. Thanks @folke!

**Reply by folke** [reply]
https://github.com/folke/noice.nvim/issues/511#issuecomment-1595272018
